### ISSUE TITLE
[a11y-defect-2]: Expand/Collapse all button causes the page to scroll

### ISSUE THREAD

**Issue by jeana-adhoc** [department-of-veterans-affairs/va.gov-team#64220]
### Point of contact

@jeana-adhoc

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

On the confirmation page for form 21-0972, when interacting to with the `Expand all +` / `Collapse all - ` button above the accordions, the following behavior happens:

Focus *DOES* remain on the button, but the page is scrolling.

https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/6a0413fb-68e6-4fef-a1ae-36baa4c6394b

### Reproduction steps

1. Navigate to [21-0972 Confirmation page](https://staging.va.gov/supporting-forms-for-claims/alternate-signer-form-21-0972/confirmation) (You may have to fill out the form with fake data, or do this in a local environment)
2. Click the `Expand all` button above the accordions
3. Observe that the page has scrolled
4. Scroll back up, see that the focus highlight/border is still on the button (which now says `Collapse all`)

### Proposed solution or next steps

The page should not scroll when the accordions expand.

I've found [one app on va.gov that has the accordions behaving as expected]( https://staging.va.gov/manage-va-debt/#other-ways-to-make-a-payment) and [one page on drupal where you can also see it behaving as expected](https://staging.va.gov/health-care/about-va-health-benefits/dental-care/)

### References, articles, or WCAG support

1. [WCAG 2.4.7 Focus Visible Level AA](https://www.w3.org/WAI/WCAG21/quickref/#focus-visible) - [VA.gov Experience standard #9](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/va-gov-experience-standards)
2. [WCAG 3.2.1 On Focus Level A](https://www.w3.org/WAI/WCAG21/quickref/#on-focus) - [VA.gov Experience standard #10](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/va-gov-experience-standards)
3.

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [X] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [X] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [X] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by Thrillberg** [comment]
@jeana-adhoc This bug is appearing because we're using the v3 of the `va-accordion` component. It sounds like the solution is to not use v3 until it's fixed in v3. Should we go ahead and not use v3 and open a ticket for this to be addressed in the design system?

**Comment by jeana-adhoc** [comment]
Hi @Thrillberg & @ndsprinkle - Are we sure it's a v3 issue? I can't recreate this in storybook - but I'm also not sure if it's because they only have 3 accordions in their example. Even though I made the screen as small as possible so that scroll would be inevitable, [the page still didn't jump when I expanded all in storybook](https://design.va.gov/storybook/iframe.html?id=uswds-va-accordion--default&viewMode=story).

**Comment by Thrillberg** [comment]
It appears to reproduce for me. Please see the attached video.
[Private Zenhub Video](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbzFnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--cc967c7840d24a2659ca9c0654548139ffb2fce6/Screen%20Recording%202023-09-01%20at%2012.00.40%20PM.mov)

**Comment by jeana-adhoc** [comment]
@Thrillberg - We have a few options I think.

1. If it's not a huge refactor lift, let's revert these components to the earlier version. 
2. 1. If it's a *HUGE* refactor lift, we can just launch with this as is.

I have [filed a bug with the DST regarding this behavior](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2060).

**Comment by Thrillberg** [comment]
@jeana-adhoc [PR here](https://github.com/department-of-veterans-affairs/vets-website/pull/25507) is merged and will be deployed shortly! I'll leave this ticket in Ready for Review until you confirm in staging and we can move it to Done.

**Comment by Thrillberg** [comment]
Also, thanks for filing the bug with DST!

**Comment by jeana-adhoc** [comment]
This looks good! Confirmed in staging! Closing ticket.
